src/Dapr.Actors.AspNetCore/WebHostBuilderExtensions.cs

```diff
@@ -22,27 +24,29 @@ public static class WebHostBuilderExtensions
         /// Configures the service to use the routes needed by Dapr Actor runtime.
         /// </summary>
         /// <param name="hostBuilder">The Microsoft.AspNetCore.Hosting.IWebHostBuilder to configure.</param>
-        /// <param name="configureActorRuntime">Adds a delegate to configure Actor runtime..</param>
+        /// <param name="runtimeOptions">Options to configure Actor runtime..</param>
```

Comment:
suggestion: All this parameter `configure` and document it as `A delegate used to register actors and configure the actor runtime.`.